edvart/report_sections/univariate_analysis.py

```diff
@@ -256,8 +256,10 @@ def histogram(
     ----------
     series : pd.Series
         Numerical series.
-    bins : int, optional
-        Number of bins of the histogram. If None, the number of bins is inferred.
+    bins : int or str or array_like, optional
+        If bins is an int, it defines the number of equal-width bins in the range of the series.
+        If bins is a string, it defines the method used to calculate the optimal bin width.
```

Comment:
I'd add a link to seaborn documentation listing the methods.

Reply:
Good point. Added a link to numpy since seaborn only says that it's passed to numpy.